src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -345,7 +345,11 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
 
   antlrcpp::Any visitValuesClause(
       SparqlAutomaticParser::ValuesClauseContext* ctx) override {
-    return visitChildren(ctx);
+    if (ctx->dataBlock()) {
+      return visit(ctx->dataBlock()).as<GraphPatternOperation::Values>();
+    } else {
+      return NULL;
```

Comment:
Please don't return a null ptr here. I hope that the `as<>` method of optional would turn this into an exception, but it it still bad. 
The semantically correct type for a `?` parsing rule would be `std::optional<Values>`. I was at first confused why this is needed at all but then I also realized that the `ValuesClause` rule is ALWAYS visited but is possibly empty...